modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/configurators/TargetPlatformRootProjectConfigurator.java

```diff
@@ -201,13 +229,28 @@ private void _configureTargetPlatformIDE(Project project) {
 			"com.liferay", "com.liferay.portal");
 	}
 
-	private String _fixBadVersionNumbers(String version) {
-		if (version.equals("7.0.6")) {
-			return "7.0.6.1";
-		}
+	private static final String _ARTIFACT_ID_RELEASE_DXP_BOM =
+		"release.dxp.bom";
 
-		return version;
-	}
+	private static final String _ARTIFACT_ID_RELEASE_DXP_BOM_COMPILE_ONLY =
+		"release.dxp.bom.compile.only";
+
+	private static final String _ARTIFACT_ID_RELEASE_DXP_DISTRO =
+		"release.dxp.distro";
+
+	private static final String _ARTIFACT_ID_RELEASE_PORTAL_BOM =
+		"release.portal.bom";
+
+	private static final String _ARTIFACT_ID_RELEASE_PORTAL_BOM_COMPILE_ONLY =
+		"release.portal.bom.compile.only";
+
+	private static final String _ARTIFACT_ID_RELEASE_PORTAL_DISTRO =
+		"release.portal.distro";
+
+	private static final String _GROUP_ID_PORTAL = "com.liferay.portal";
```

Comment:
Please update the variable name so its consistent with the other `static` `final` variables in this class: `_GROUP_ID_COM_LIFERAY_PORTAL`